include/CMakeLists.txt

```diff
@@ -28,8 +26,6 @@ if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.19")
 	)
 endif()
 
-# NOTE: Because Vulkan::Headers header files are exposed in the public facing interface
-# we must expose this library as public to users.
```

Comment:
Removed documentation in favor of test code where I now check INTERFACE properties.